src/runtime/hexagon_remote/sim_host.cpp

```diff
@@ -309,6 +317,18 @@ int halide_hexagon_remote_run(handle_t module_ptr, handle_t function,
         printf("Hexagon simulator executed function 0x%x in %d cycles\n", function, cycles);
     }
 
+    if (getenv("HL_HEX_SIM_STATS")) {
+        HEXAPI_Status status = HEX_STAT_SUCCESS;
+        char Buf[4096];
+        status = sim->EmitPerfStatistics(0,0,0,0,Buf, sizeof(Buf));
+        if (status != HEX_STAT_SUCCESS) {
+            printf("HexagonWrapper::EmitStatistics failed: %d\n", status);
+        } else {
+            // Print out the full stats
+            printf("%s\n", Buf);
```

Comment:
Is there a way to reset the stats prior to running the pipeline? This is going to print out the stats from all past simulated code, even if many pipelines have run previously.